mlrun/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -80,7 +79,7 @@ def create_model_endpoint(
         :param db_session:             A session that manages the current dialog with the database.
         :param model_endpoint:         Model endpoint object to update.
 
-        :return: Model endpoint object.
+        :return: ModelEndpoint object.
```

Comment:
When referring to python objects like classes, variables and arguments always add back-quotes like so: 
```suggestion
        :return: `ModelEndpoint` object.
```